src/containers/AddPost/AddPost.js

```diff
@@ -30,4 +51,17 @@ class AddPost extends Component {
     }
 }
 
-export default AddPost;
\ No newline at end of file
+const mapStateToProps = state => {
+    return {
+        posts: state.posts.posts,
+        loading: state.loading
```

Comment:
Can you please check this as we have two reducers and state should be taking from one of them
state.loading